sklearn/linear_model/coordinate_descent.py

```diff
@@ -249,6 +249,17 @@ def lasso_path(X, y, eps=1e-3, n_alphas=100, alphas=None,
     [[ 0.          0.          0.46915237]
      [ 0.2159048   0.4425765   0.23668876]]
 
+    References
+    ----------
+    R. Tibshirani (1996) "Regression shrinkage and selection via the lasso"
+    J. Royal. Statist. Soc B., Vol. 58, No. 1, pages 267-288).
+
+    J. Friedman, T. Hastie, R. Tibshirani (2010) "Regularization Paths for
+    Generalized Linear Models via Coordinate Descent"
+    Journal of Statistical Software, Vol. 33, No. 1, pages 1-22.
+
+    O. Fercoq, A. Gramfort, J. Salmon (2015) "Mind the duality gap: safer
+    rules for the Lasso" Proc. ICML. http://arxiv.org/pdf/1505.03410v1
```

Comment:
I think the scikit-learn project might want to enforce to automatically reject pull requests for methods with poor puns in the title of their reference paper.